docs/user-guide/nextcloud-integration/README.md

```diff
@@ -69,7 +69,61 @@ In addition to listing files attached to a work package, the **Files** tab now a
 
 ![OP_linked_files](1_1_00-All_files_available.png)
 
-> **Note:** In this version of the integration, you can only link files to work packages on Nextcloud; adding a new link to a Nextcloud file via the OpenProject interface is not yet possible, but will be possible in the near future.
+
+To link a Nextcloud file to the current work package, you can either:
+
+- select a file from your computer, which will be uploaded to Nextcloud and linked to this work package
+- select an existing file in Nextcloud to link to
+
+#### Upload and link file(s)
+
+If the file you want to link has not yet been uploaded to Nextcloud, you can do so by clicking on the **Upload files** link. 
+
+>> IMG: Focus on Upload file link
+
+You will then be prompted to select a file (or multiple files) on your computer that you want to upload to Nextcloud. 
+
+>> IMG: Local browser file picker
+
+Alternatively, you can also simply drag a file or folder on your computer on to this area (under the name of your Nextcloud file storage) and drop it in the drop zone that appears.
```

Comment:
@psatyal maybe change "file or folder on your computer on to this area" to "file or folder from your computer on to this area"